CONTRIBUTING.md

```diff
@@ -2,14 +2,6 @@
 
 Contributions should follow the [principals](../README.md#principals) of rules_scala_annex.
 
-## Documentation
```

Comment:
I believe this will conflict with your other PR to bring back doc.

Reply:
Yup, I'll need to re-add this section. Thanks for noticing that.